raft-tech/TANF-app
# Increase memory to resolve file upload errors

## Summary of Changes
Pull request closes #2350 _

Deployed memory increased from 512M to 2G, which resolves upload errors. Additionally, when files are missing name (see [ref](https://github.com/raft-tech/TANF-app/pull/1918#pullrequestreview-1108252379)), the log model throws error because filename is null. Changed the assigned filename to string 'None'.

## How to Test
Memory is not limited, so the error should not be happening locally. To recreate memory error locally, we should use:
```
    mem_limit: 256m
    deploy:
      resources:
        limits:
          memory: 220M
```
in docker compose.

When deployed,

1. Open http://localhost:3000/ and sign in.
1. Proceed with submitting files
1. make sure the files are uploaded to SFTP server and logs are not throwing any error.
1. Login into admin page, look at the upload logs and the files.

## Deliverables

Generally, files should be uploaded without any issue.

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] Test files are uploaded and there is no error in the logs

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
